### ISSUE TITLE
Github project for sync related issues

### ISSUE THREAD

**Issue by JulienFS** [nextcloud/server#45470]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Is your feature request related to a problem? Please describe.**
I would like to invest some time to fix problems related to file synchronization, however all I have is 2k issues to browse with most of them dating from 2018. Since it's a tough cookie that would be nice to have some sort of coordinated effort under some core team supervision.

**Describe the solution you'd like**
I wish there were a github project tracking everything related to file synchronization (and to some extent everything sync related like calendars) orchestrated by some Nextcloud knowledgable people.

**Describe alternatives you've considered**
At this point I'm considering using the same tools for product management as at work, however they are not open-source oriented and they would be only for my personal use, which is not that nice to leverage the community's power.

**Additional context**
Sorry to use an issue for that but this is code related and I couldn't find any place to talk about this matter somewhere else.

**Comment by kesselb** [comment]
Hi, 

I will forward your issue, not sure if we have something like that or can provide it.

You may want to filter for issues with the label "1. To develop" as those issues are usually "confirmed" and then sort by comments or reactions. 

We have a developer section in our forums: https://help.nextcloud.com/c/dev/11

Some more information: https://nextcloud.com/developer/

**Comment by joshtrichards** [comment]
Also the vast majority of open issues are labeled by feature. So `feature: dav` and `feature: caldav` are good candidates in the Server repo to filter by given your topic interest.

Depending on what you mean by synchronization, there are also other repos for things like the Desktop, Android, and iOS clients. Also the server repo does use a couple other labels to indicate mobile/desktop client involvement.

Though I'm assuming part of what you're getting at is: "Hey, is there a Kanban prioritization board somewhere?" Or something along those lines. :-)

**Comment by JulienFS** [comment]
Hi, thanks for these answers. Indeed there are some ways to navigate within issues. I didn't know about the dev section of the forum, thanks for that too.

> Though I'm assuming part of what you're getting at is: "Hey, is there a Kanban prioritization board somewhere?" Or something along those lines. :-)

Yes ^^ I would like to have a coherent approach to the sync problems I've encountered and dug the code for (I reported my exploration in an issue about checksums). Which means I need an overview of the whole situation. Things like collecting issues, scrapping comments for technical details, desired behavior, they take time. I'm willing to do so but first I need to know if these things already exist somewhere, and if they don't I need to know where to put them. In any case they should be easy to find for newcomers.
All this work has value, especially because it cost precious time. I don't really like the idea that we have to do our own brewing on our own side.

Since github offers "projects" and it requires no extra tool or setup I tend to think it should be used more extensively.

If you don't have anything like this at this point maybe I should setup something on a fork project and show you what I mean :thinking:

**Comment by JulienFS** [comment]
> > Since github offers "projects" and it requires no extra tool or setup I tend to think it should be used more extensively.
> 
> https://github.com/orgs/nextcloud/projects?query=is%3Aopen

Thank you, I had only looked at the server's project page. At least it gives some information about what the team is working on (and by extension, what is not being worked on).

I created a project to track my progress on my fork : https://github.com/users/JulienFS/projects/1. I leave the link here for the record.  If anyone passing by wants to get involved feel free to ping me.
